src/components/Typography/Link.tsx

```diff
@@ -36,6 +36,7 @@ function Link({
       target={target}
       rel="noopener noreferrer"
       style={style}
+      tabIndex={0}
```

Comment:
sorry, can you also remove this

Reply:
Yes, sorry - I had thought it was still needed for focusing, but not the case. Removed